akirachix/Upile-Informational
# Feature/mission vision

**What is this?**
        This PR updates the Mission, Vision, and Testimonies components to enhance alignment, responsiveness, and styling according to the latest design requirements.
    **Why this?**
        To improve the user interface by ensuring consistent styling and alignment across different screen resolutions and devices.
    **How are you doing this?**
        List of updates for features, enhancements, or bugfixes:
            Adjusted the alignment and spacing of the Mission and Vision components to ensure uniform width and alignment.
  
**Type of change**

**Specify the kind of PR/MR**

Please delete options that are not relevant.


    Improvement

**How can it be tested?**

    **Mission and Vision Components:**
        Verify that the width and alignment of the Mission and Vision boxes are consistent.
        Check for responsiveness on different screen sizes to ensure proper display.

    **Testimonies Component:**
        Test responsiveness on various devices 
        Ensure that the layout adjusts appropriately without disrupting the original design.
        Confirm that the image and text align correctly and the spacing between testimony boxes is increased as per 
        requirements.

**Screenshot**
![image](https://github.com/user-attachments/assets/85f6aeba-af66-4712-aa7f-61abb8c1c6c0)
![image](https://github.com/user-attachments/assets/a5b41e14-e138-4e0f-9b55-9cd148091687)



**Where is it documented?**
https://wambuinatalygakenye.atlassian.net/browse/UP-119
